devices/paulmann.js

```diff
@@ -108,4 +108,11 @@ module.exports = [
             'on', 'off', 'toggle', 'brightness_step_up', 'brightness_step_down', 'color_temperature_move', 'color_move', 'brightness_stop',
             'brightness_move_down', 'brightness_move_up', 'color_loop_set', 'enhanced_move_to_hue_and_saturation', 'scene_*'])],
     },
+    {
+        zigbeeModel: ['RGBCW_LIGHT'],
+        model: '501.27',
+        vendor: 'Paulmann',
+        description: 'Paulmann Light',
+        extend: extend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please also add the color temp range as described in https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature